src/MANYBODY/pair_eam.cpp

```diff
@@ -230,6 +234,7 @@ void PairEAM::compute(int eflag, int vflag)
     p = MIN(p,1.0);
     coeff = frho_spline[type2frho[type[i]]][m];
     fp[i] = (coeff[0]*p + coeff[1])*p + coeff[2];
+    count_embed[i] = 0;
```

Comment:
I suggest moving this line to be right before the jj loop
further down where count_embed is incremented for each i,j pair
within the cutoff

Reply:
done